autoload/JuliaFormatter.vim

```diff
@@ -2,6 +2,10 @@ if get(g:, 'JuliaFormatter_loaded')
     finish
 endif
 
+if !exists("g:JuliaFormatter_options")
+  let g:JuliaFormatter_options = ''
+endif
+
```

Comment:
I think these lines should be in `plugin/JuliaFormatter.vim`.

Reply:
Fixed